website/docs/developer_guide.md

```diff
@@ -103,6 +132,26 @@ multiple adapter types
 - `SNPCoverageAdapter` - this adapter takes a `BamAdapter` or `CramAdapter` as a
   subadapter, and calculates feature coverage from it
 
+### Displays
+
+A display is a method for displaying a particular track in a particular view
+
+For example, we have a notion of a synteny track type, and the synteny track
+type has two display models
+
+- DotplotDisplay, which is used in the dotplot view
```

Comment:
Should we link to a source code example of displays being used?

Reply:
I think having the names of the displays may be a good starting point for a reader to find what is needed, but if you think it would help we can add an example too

Reply:
I think this should be good. We can start with the names. If we see down the road that users get confused then we can try adding an example.